### PR TITLE
feat(click-stealer): show cursor not-allowed

### LINKED ISSUE #25: Show cursor type `not-allowed` when keyboard-only is active
See: https://developer.mozilla.org/en/docs/Web/CSS/cursor

### PR DESCRIPTION
**Pull request by reinoute**
Resolves #25

### CODE REVIEW

File: src/assets/a11y-viewer.css [hunk 1]
```diff
@@ -222,8 +222,7 @@ main {
 }
 
 main.is-disabled {
-    pointer-events: none;
```
Comment (jbmoelker) [inline src/assets/a11y-viewer.css:225]: @reinoute Why is this also removed? Looks unrelated to this PR. And I'd expect `pointer-events` to be `none`. As that's exactly what the feature aims to do "keyboard only" is really "no pointers".

File: src/assets/a11y-viewer.css [hunk 2]
```diff
@@ -236,6 +235,7 @@ main.is-disabled:before {
     bottom: 0;
     left: 0;
     z-index: 1;
+    cursor: not-allowed;
```
Comment (jbmoelker) [inline src/assets/a11y-viewer.css:238]: this looks good.

File: src/assets/a11y-viewer.css (hunk 1 above)
Comment (reinoute) [inline src/assets/a11y-viewer.css:225]: The :before layer already blocks the clicks.
I tested this, and the `pointer-events: none` doesnt seem to have any value anymore.
